query-engine/launch.sh

```
@@ -0,0 +1 @@
+docker compose build && docker compose up -d && docker compose logs -f llamaindex
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)